src/scss/layout/_error-pages.scss

```diff
@@ -0,0 +1,91 @@
+.error-page {
```

Comment:
There is no need for this nesting. You're already taking care of it with the namespacing of BEM. Please do not add extra specificity where it's not needed.

Reply:
Ah good catch! This was actually a mistake :D only the .no-sidebar .main-content was supposed to be nested. Fixed.